Doc/library/os.rst

```diff
@@ -5153,14 +5153,21 @@ Miscellaneous System Information
    .. availability:: Unix.
 
 
-.. function:: cpu_count()
+.. function:: cpu_count(*, usable=False)
 
-   Return the number of CPUs in the system. Returns ``None`` if undetermined.
+   Return the number of logical CPUs in the system. Returns ``None`` if
```

Comment:
You have to update the documentation that it will return the number of logical CPUs for the process if the usable is True.

Reply:
The doc says:

> If *affinity* is true, return the number of logical CPUs the current process can use.

It's not clear enough? Do you want to propose a different phrasing?

Reply:
That is reasonable.

I'd add "this may be less than the number of logical cpus returned by affinity=False due to OS or container limitations imposed upon the process" to make it more clear _why_ people should want to use the `affinity=True` argument.

PS thanks for making it keyword only!

I do wish this API never used the term "cpu"... everything these days is really a "logical_core" and what that even means depends a lot on underlying infrastructure and platform that Python may not be able to introspect.  Way too late for that though. :)

Reply:
My PR adds "logical CPU" to the doc. In previous bug reports, I saw come confusion between physical CPU core, CPU packages, CPU threads, and logical CPUs.